docs/tutorials/alert-remediation.rst

```diff
@@ -44,7 +62,7 @@ Alerts can also be remediated with bash commands:
           alert_name: SomeAlert
       actions:
       - node_bash_enricher:
-         bash_command: do something
+        bash_command: do something
```

Comment:
@pavangudiwada did we remove necessary whitespace here?

Reply:
Yes, fixed it thanks.